### DISCUSSION TITLE
I have a 500Gb disk image and I'd like to use IPED to search for cryptocurrencies related content (wallets, private keys)

### DISCUSSION THREAD

**Discussion by TheGigaQuant** [sepinf-inc/IPED#2363] [Q&A] [answered]
Hey there, I have an HDD I made a fulldisk image and I tried to use IPED without success.
When I open IPED with my image, it says something like +60,000 hours and increasing, can't make it work.
Question : How can I use IPED only to find cryptocurrencies related stuff ?
Why is it taking so much hours to be read ?

The image is on a NVME SSD, super fast one, and the CPU seems to be low used while using IPED.
Thanks !

**Comment by lfcnassif** [comment]
Hi. We need more information to help you. What was your command line? Could you share the processing log? What is the image main file system? Could you print a screenshot of the whole processing Window, without scrollbars, and post it here?

**Comment by TheGigaQuant** [comment]
Hi !
I'm using the IPED 4.1.6 on Windows 11.
The image is a whole harddrive copy, bit-by-bit using HDDSuperTool since the harddrive had some issues.
If I remember well, I have Windows 7 installed on this HDD but I also have an old Windows XP system.

Here is the command line I'm using : 
`.\iped.exe -d D:\MyImage`

![image](https://github.com/user-attachments/assets/c8303ff8-6163-4257-8c06-0e0958f421d0)

[IPED-2024-11-21-12-21-03.log](https://github.com/user-attachments/files/17844856/IPED-2024-11-21-12-21-03.log)

Thank you for your help and support ♥

**Reply by wladimirleite** [reply, marked as answer]
It is a plain disk image, right?
You have to add an extension .dd or .001.
Otherwise IPED will treat it as a huge unknown file.

**Reply by TheGigaQuant** [reply]
Damn, it was that simple lol, thanks !

**Reply by TheGigaQuant** [reply]
Once the scan is done, and I opened the IPED-SearchApp.exe, on under what category should I find any cryptocurrencies related items ?
Thanks :)

**Reply by wladimirleite** [reply]
Ideally you should create a custom processing profile, to enable only the options that you need.
I think with the default profile you should be able to find items in Metadata tab (bottom left), then selecting group "Regex" and finally filetring the cryptocurrency related properties.
Another option is to type "Regex:CR" and then press [TAB] (one or more times) in search field, so properties related to cryptocurrencies will be displayed.
Filters like "Regex\:CRIPTOCOIN\_BITCOIN\_ADDRESS:*" should work.
You can check the defined regular expressions in "conf\RegexConfig.txt" file. You can also define your own expressions there (you would need to process the evidence again to use a new configuration).

**Reply by mobab-th** [reply]
> Once the scan is done, and I opened the IPED-SearchApp.exe, on under what category should I find any cryptocurrencies related items ? Thanks :)

If a hardware wallet is used, you could have a bookmark “Possible use of hardware crypto wallets” and metadata with information about the manufacturer and product.

**Reply by TheGigaQuant** [reply]
Idk why I don't have CRIPTOCOIN_BITCOIN_ADDRESS in regex
Only the following are shown :
![image](https://github.com/user-attachments/assets/c67ab2a9-c93c-4dc1-a3f7-9b37622aedad)

I checked the RegexConfig.txt file and it's uncommented, so idk

**Reply by paulobreim** [reply]
If it doesn't appear, it's because no bitcoin address was found. Are you sure it should be there?

**Reply by TheGigaQuant** [reply]
Oh ok, I didn't knew it appears only if matching elements were found.
Yes, I got it on another image to test it.
Thank you very well for your help ♥
